src/hotspot/share/utilities/exceptions.cpp

```diff
@@ -608,4 +610,18 @@ void Exceptions::log_exception(Handle exception, const char* message) {
                          MAX_LEN, exception->print_value_string(),
                          MAX_LEN, message);
   }
+  maybe_log_call_stack();
+}
+
+void Exceptions::maybe_log_call_stack() {
+  LogStreamHandle(Info, exceptions, stacktrace) st;
+  if (st.is_enabled()) {
+    Thread* t = Thread::current_or_null();
+    if (t != nullptr && t->is_Java_thread()) { // sanity
```

Comment:
Do we need this? If we just rely on assertions then all we need is to call `JavaThread::current`.

Reply:
Fixed.